templates/cassiopeia/error.php

```diff
@@ -27,7 +27,7 @@
 $itemid   = $app->input->getCmd('Itemid', '');
 $sitename = $app->get('sitename');
 $menu     = $app->getMenu()->getActive();
-$pageclass = $menu->params->get('pageclass_sfx');
+$pageclass = $menu ? $menu->params->get('pageclass_sfx') : '';
```

Comment:
This is correct, but I prefer it more clearly
```suggestion
$pageclass = $menu !== null ? $menu->params->get('pageclass_sfx', '') : '';
```